src/actions/actionHistory.tsx

```diff
@@ -62,6 +62,7 @@ type ActionCreator = (history: History) => Action;
 
 export const createUndoAction: ActionCreator = (history) => ({
   name: "undo",
+  trackEvent: { category: "history" },
```

Comment:
Any specific reason we want to track `undo/redo`?

Reply:
I'd like to know how the UI buttons for undo/redo are used. Not sure if tracking it this way will be useful, but we can at least do `{uiCount}/{keyboardCount}` to get a rough idea?

Reply:
though maybe it'll be too many events